specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+        "jars": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "pyFiles": {
+          "x-ms-client-name": "pythonFiles",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "files": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "driverMemory": {
+          "type": "string"
+        },
+        "driverCores": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "executorMemory": {
+          "type": "string"
+        },
+        "executorCores": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "numExecutors": {
+          "x-ms-client-name": "executorCount",
+          "format": "int32",
+          "type": "integer"
+        },
+        "archives": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "queue": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "conf": {
+          "x-ms-client-name": "configuration",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SparkSessionCollection": {
+      "type": "object",
+      "properties": {
+        "from": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "total": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "sessions": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SparkSessionJob"
+          }
+        }
+      }
+    },
+    "SparkSessionJob": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "appId": {
+          "type": "string"
+        },
+        "owner": {
+          "type": "string"
+        },
+        "proxyUser": {
+          "type": "string"
+        },
+        "kind": {
+          "type": "string"
+        },
+        "log": {
+          "x-ms-client-name": "logLines",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "state": {
+          "type": "string"
+        },
+        "appInfo": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SparkSessionJobRequest": {
+      "type": "object",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SessionJobKind",
+            "modelAsString": false
```

Comment:
Recommendation is `true`, since locking the enum might create compatiblity problem with future, while using `true` means "extensible enum", so you still get an enum but it's non breaking in you add a format one day (even if in any months)

Reply:
Fixed it. Thanks for your comment.